sources/packages/web/nginx.conf

```diff
@@ -13,4 +13,10 @@ http {
   default_type  application/octet-stream;
   sendfile        on;
   keepalive_timeout  65;
+  server {
```

Comment:
what about Gzip compression settings which is mentioned in the ticket (from rocket chat)?

Reply:
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/2406 created for the validation whether the setting is needed or not